sdcm/microbenchmarking.py

```diff
@@ -78,7 +78,7 @@ def check_regression(self, current_results, html_report_path):
         #           }
         # }
         allowed_stats = ("Current", "Last (commit id)", "Diff last [%]", "Best (commit id)", "Diff best [%]")
-        metrics = ("aio", "frag/s", "cpu", "time (s)")
+        metrics = ("aio", "frag/s", "cpu", "time (s)", "mad f/s", "max f/s", "min f/s")
```

Comment:
Take out the new metrics from this PR.

Reply:
Also take out "cpu" and "time (s)" we want to stop reporting them.

Reply:
removed.